### DISCUSSION TITLE
Mini game Kraken

### DISCUSSION THREAD

**Discussion by Ario8260** [sunflower-land/sunflower-land#3118] [Ideas]
I have a suggestion for this mini game! Add an option to get special fish or octopuses by paying more resources without playing the game
for example 50 eggplants or 1 gold or….
There are people who play with illnesses or disabilities and such a mini-game is more difficult to complete

thanks in advance

**Comment by RezA7X** [comment]
It is a really good and interesting suggestion.

**Comment by sabryder** [comment]
Great idea
People can choose whatever they want
If they cant pass that mini game they can skip that with burning extra resources😍🌻🍻
